src/client/sandbox/node/document/index.js

```diff
@@ -235,5 +236,12 @@ export default class DocumentSandbox extends SandboxBase {
             },
             setter: null
         });
+
+        overrideDescriptor(window[nativeMethods.documentCookiePropOwnerName].prototype, 'cookie', {
```

Comment:
It's difficult to read. Let's move the prototype condition to separate variable:
```js
const documentCookiePropOwnerPrototype = window[nativeMethods.documentCookiePropOwnerName].prototype;

overrideDescriptor(documentCookiePropOwnerPrototype, 'cookie', {...});
```

Do this for `cookie` and `all` properties.